adapter/internal/adapter/adapter.go

```diff
@@ -139,6 +143,8 @@ func runManagementServer(server xdsv3.Server, enforcerServer wso2_server.Server,
 		if err = auth.Init(); err != nil {
 			logger.LoggerMgw.Error("Error while initializing autherization component.", err)
 		}
+		logger.LoggerMgw.Debug("Updating health status of AuthService")
+		health.AuthService.SetStatus(err == nil)
```

Comment:
Remove this check, AuthService

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1899/commits/c8609cb566c166e551fe02c44164cfdfbe1611d4